runtime/etc/Makefile.include

```diff
@@ -58,6 +58,7 @@ endif
 COMP_GEN_CFLAGS = $(MAKE_COMP_GEN_CFLAGS) $(COMP_GEN_USER_CFLAGS)
 
 include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.threads-$(CHPL_MAKE_THREADS)
+-include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.locale-$(CHPL_MAKE_LOCALE_MODEL)
```

Comment:
I think I would be more specific here and instead of just "locale" I'd use "locModel" (but probably not the more wordy "locale-model". The terms "locale" and "locale model" are definitely very different things in Chapel.

The filename of runtime/etc/Makefile.locale-gpu would have to change correspondingly, of course.